kafka-bridge.go

```diff
@@ -188,8 +188,16 @@ func (bridge BridgeApp) forwardMsg(kafkaMsg string) error {
 		return err
 	}
 
+	tid, err := extractTID(kafkaMsg)
+
+	if err != nil {
+		fmt.Printf("Error parsing transaction id: %v\n", err.Error())
+		tid = "tid_" + uniuri.NewLen(10) + "_kafka_bridge"
+		fmt.Printf("Generating tid: " + tid)
+	}
+
 	req.Header.Add("X-Origin-System-Id", "methode-web-pub") //TODO: parse this from msg
```

Comment:
You should read Origin-System-Id from the kafka message. Same for transactionId.